src/lib/CalibrationVerification.cpp

```diff
@@ -989,9 +998,17 @@ void CalibrationVerification::PrintErrorsSummary() {
   }
   norms_averaged = norms_summed / camera_errors_opt_.size();
 
+  float angular_summed, angular_averaged;
+  for (int i = 0; i < camera_angular_errors_.size(); i++) {
+    angular_summed += camera_angular_errors_[i];
+  }
+  angular_averaged = angular_summed / camera_angular_errors_.size();
+
   file << "\n-----------------------------------------------------------\n\n"
        << "Outputting Error Statistics for Optimized Camera Calibrations:\n"
        << "Average Error Norm (pixels): " << norms_averaged << "\n"
+       << "Average Error Norm (rad): " << angular_averaged << "\n"
+       << "Average Error Norm (degree): " << angular_averaged*57.2958 << "\n"
```

Comment:
use this function instead: https://github.com/nickcharron/vicon_calibration/blob/master/include/vicon_calibration/Utils.h#L111

Reply:
Yes